docs/deploy/azure.md

```diff
@@ -15,54 +15,58 @@ next:  ["Import data into your installation", "../howto/import.html"]
 ⏰ Expected deployment time: 25 min
 {: .note }
 
-{% include toc.html %}
+## Overview
 
-{% include_relative includes/prerequisites.md %}
+lakeFS has several dependencies for which you need to select and configure a technology or interface: 
 
-## Supported storage types
+![](./deploy-on-azure.excalidraw.png)
```

Comment:
maybe personal preference :) I found the previous page less clear because it was a wall of text, and figured the image would give the user a "map" of what was the come. WDYT?

Reply:
Consider putting this in the index page rather than here. Otherwise I would expect it to appear on each of the deployment pages.
Also remove the overview from TOC

Reply:
Done

Reply:
I'll add the diagram to other deployment pages in a separate PR